docs/modules/user-guide/partials/proc_migrating-commands.adoc

```diff
@@ -2,26 +2,26 @@
 = Migrating commands
 
 [role="_abstract"]
-This section describes how to migrate plug-ins to devfile v2. There ARE major changes in the `commands` section of the devfile. A command specified in a v1.0 devfile will NOT work as it is in a devfile v2.0. 
+This section describes how to migrate commands to devfile v2. There ARE major changes in the `commands` section of the devfile. A command specified in a v1.0 devfile will NOT work as it is in a devfile v2.0.
```

Comment:
We should avoid mentioning the v2 minor version as this doc should apply to any v2.x
```suggestion
This section describes how to migrate commands to devfile v2. There ARE major changes in the `commands` section of the devfile. A command specified in a v1.0 devfile will NOT work as it is in a devfile v2.x.
```